.github/workflows/test.yml

```diff
@@ -0,0 +1,66 @@
+name: Scala tests with coverage
+
+on:
+  pull_request:
+    paths-ignore: ['**.md']
+  push:
+    paths-ignore: ['**.md']
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
```

Comment:
note that this is a change from Travis (which used Trusty) ... I think it's a good move but calling it out in case we see any strange behavior.